displaytag/pom.xml

```diff
@@ -198,7 +198,7 @@
     <dependency>
       <groupId>net.sourceforge.htmlunit</groupId>
       <artifactId>neko-htmlunit</artifactId>
-      <version>2.67.0</version>
+      <version>2.70.0</version>
```

Comment:
@allantodd Won't work past 2.67.0, please go back to that.  Only way newer will work is figuring out 1 class in httpunit that I've been unsuccessful figuring out.  htmlunit project there internalized a lot of xerces so things that did work don't now.

Reply:
done